android-client-sdk/proguard-rules.pro

```diff
@@ -18,4 +18,6 @@
 
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
-#-renamesourcefileattribute SourceFile
\ No newline at end of file
+#-renamesourcefileattribute SourceFile
+-keep class kotlin.Metadata { *; }
```

Comment:
/is this necessary

Reply:
Im not sure but its recommended in the jackson docs